sdk/openai/azure-ai-openai/src/test/java/com/azure/ai/openai/models/ChatCompletionsToolCallUnitTests.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.openai.models;
+
+import com.azure.core.util.BinaryData;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Collections;
+
+/**
+ * Unit tests for ChatCompletionsToolCall.
```

Comment:
I would add a brief word in this part of the documentation explaining why these tests are necessary for this particular asymmetry between serialization and deserialization.